src/jetstream/.realize.yaml

```diff
@@ -0,0 +1,21 @@
+settings:
```

Comment:
Would be good to add some docs to the jetstream dev guid that covers how to use this